src/features/ITGovernance/Admin/GRBReview/DecisionRecord/_components/DecisionRecordTable.tsx

```diff
@@ -0,0 +1,178 @@
+import { Column, useSortBy, useTable } from 'react-table';
+import { Button, ModalHeading, Table } from '@trussworks/react-uswds';
+import { SystemIntakeGRBReviewerFragment } from 'gql/generated/graphql';
+
+import Modal from 'components/Modal';
+import {
+  currentTableSortDescription,
+  getColumnSortStatus,
+  getHeaderSortIcon
+} from 'utils/tableSort';
+
+import GRBReviewerVote from './GRBReviewerVote';
+
+type DecisionRecordTableProps = {
+  grbReviewers: SystemIntakeGRBReviewerFragment[];
+};
+
+/**
+ * Displays the decision record table for GRB reviews
+ */
+const DecisionRecordTable = ({ grbReviewers }: DecisionRecordTableProps) => {
+  const { t } = useTranslation('grbReview');
+
+  // Set GRB reviewer to view comment in modal
+  const [grbReviewerViewComment, setGRBReviewerViewComment] =
+    useState<SystemIntakeGRBReviewerFragment | null>(null);
+
+  /** Columns for table */
+  const columns = useMemo<Column<SystemIntakeGRBReviewerFragment>[]>(() => {
+    return [
+      {
+        Header: t<string>('participantsTable.name'),
+        id: 'commonName',
+        accessor: ({ userAccount }) => userAccount.commonName,
+        width: 'auto'
+      },
+      {
+        Header: t<string>('participantsTable.votingRole'),
+        accessor: 'votingRole',
+        Cell: cell => {
+          const { value } = cell;
+          return <>{t<string>(`votingRoles.${value}`)}</>;
+        },
+        width: 160
+      },
+      {
+        Header: t<string>('participantsTable.grbRole'),
+        accessor: 'grbRole',
+        Cell: cell => {
+          const { value } = cell;
+          return <>{t<string>(`reviewerRoles.${value}`)}</>;
+        },
+        width: 350
+      },
+      {
+        Header: t<string>('Vote'),
+        accessor: grbReviewer => (
+          <GRBReviewerVote
+            grbReviewer={grbReviewer}
+            setGRBReviewerViewComment={setGRBReviewerViewComment}
+          />
+        ),
+        width: 350
+      }
+    ];
+  }, [t]);
+
+  const table = useTable(
+    {
+      columns,
+      data: grbReviewers,
+      autoResetSortBy: false,
+      autoResetPage: true,
+      initialState: {
+        sortBy: useMemo(() => [{ id: 'commonName', desc: false }], [])
+      }
+    },
+    useSortBy
+  );
+
+  const { getTableBodyProps, getTableProps, headerGroups, prepareRow, rows } =
+    table;
+
+  return (
+    <>
+      {
+        // View comment modal
+        grbReviewerViewComment?.voteComment && (
+          <Modal
+            isOpen={!!grbReviewerViewComment}
+            closeModal={() => setGRBReviewerViewComment(null)}
+            className="easi-modal__content--narrow"
+          >
+            <ModalHeading className="margin-bottom-05">
+              {t('decisionRecord.grbComment')}
+            </ModalHeading>
+            <GRBReviewerVote grbReviewer={grbReviewerViewComment} />
+            <p>{grbReviewerViewComment.voteComment}</p>
+          </Modal>
+        )
+      }
```

Comment:
This height for this modal on mobile takes up the entire height of the screen.  `height: 'auto'` or `height: 'fit-content'` should work here